frontend/app/components/create-badge.js

```diff
@@ -10,16 +10,13 @@ export default Component.extend({
     EmberObject.$(window).bind('resize scroll', function(e) {
       var card = EmberObject.$(document)[0].getElementsByClassName('six wide column preview');
       if (card.length != 0) {
-        if (e.type === 'scroll' && this.scrollY >= 137 && this.window.innerWidth >= 764 && !(EmberObject.$(window).scrollTop() + EmberObject.$(window).height() > (EmberObject.$(document).height() - 100))) {
+        if (e.type === 'scroll' && this.scrollY >= 137 && this.window.innerWidth >= 764 && !(EmberObject.$(window).scrollTop() + EmberObject.$(window).height() > (EmberObject.$(document).height()))) {
           card[0].style.position = 'fixed';
           card[0].style.right = '0';
           card[0].style.top = '0';
-          EmberObject.$(document)[0].getElementsByClassName('bgImg ui raised segment')[0].style.height = '600px';
-        } else if (e.type === 'scroll' && this.window.innerWidth >= 764 && (EmberObject.$(window).scrollTop() + EmberObject.$(window).height() > (EmberObject.$(document).height() - 100))) {
-          card[0].style.position = 'fixed';
           EmberObject.$(document)[0].getElementsByClassName('bgImg ui raised segment')[0].style.height = '530px';
         } else {
-          EmberObject.$(document)[0].getElementsByClassName('bgImg ui raised segment')[0].style.height = '100%';
+          EmberObject.$(document)[0].getElementsByClassName('bgImg ui raised segment')[0].style.height = '530px';
```

Comment:
These changes are not required.

Reply:
There is no effect of removing this code.